index.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     "jsx-a11y/no-static-element-interactions": "warn",
     "jsx-quotes": ["error", "prefer-single"],
     "no-alert": "off",
+    "no-console": "error",
```

Comment:
I'm betting the person who pulls in the latest config here will need to fix all of these new errors or silence the intentional ones. Since the version bump here is a "patch", do our other repos auto-pull this in?

Reply:
I will pull this into the client repos.  I'll confirm whether or not it will be auto-pulled in, but I'd imagine it's going to be manually (update the version specified in `package.json`)